### PR TITLE
Change the password form order

### PR DESCRIPTION
**Pull request by mayaeh**
Change the order of password change form on Preferences -> Security page.

I think that in the many password change form, the current password -> new password is in order.

With this change, I think that we can reduce the case of failing to change the password.

Also fixes the problem where translation of "New password" is not applied to label.

Before:

After:

### CODE REVIEW

File: app/javascript/mastodon/locales/ja.json [hunk 1]
```diff
@@ -321,7 +321,7 @@
   "status.show_less_all": "全て隠す",
   "status.show_more": "もっと見る",
   "status.show_more_all": "全て見る",
-  "status.show_thread": "続きを読む",
+  "status.show_thread": "スレッドを表示",
```
Comment (TheKinrar) [inline app/javascript/mastodon/locales/ja.json]: This is not related to the PR.

Comment (mayaeh) [reply app/javascript/mastodon/locales/ja.json]: You're right.
Thanks.

**Review by Gargron — APPROVED** [review]
